README.md

```diff
@@ -228,6 +228,42 @@ http.createServer(function (req, res) {
 
 **[Back to top](#table-of-contents)**
 
+#### Forward POST requests with a json body
+
+```ts
```

Comment:
`import { IncomingMessage, ServerResponse } from 'http';
import * as httpProxy from 'http-proxy';`

add this to the first